packages/ui-extensions/src/surfaces/point-of-sale/components/TextField/TextField.ts

```diff
@@ -30,7 +30,72 @@ export type EmbeddedElementProps =
   | SuccessProps
   | PasswordProps;
 
-export interface NewTextFieldProps extends InputProps {}
+export interface NewTextFieldProps extends InputProps {
```

Comment:
Instead of duplicating the InputProps here, does it also work if you fix the comments in `InputField.ts` instead?